README.md

```diff
@@ -6,5 +6,13 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails exists as a web framwork whose purpose is to allow us to use an MVC architecture to receive requests from the front end of a web app, aka the client, via an ajax request, and translate that request to ruby files through controllers and models, which send the data request to a database for either Create, Read, Update, or Destroy data and then work with those models and controllers to send the server response back to the client. I think?
```

Comment:
Not always through an AJAX request - Rails apps can also respond with full  HTML pages.